cfme/tests/v2v/test_v2v.py

```diff
@@ -2,19 +2,20 @@
 import pytest
 
 from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.log import logger
 
-pytestmark = [pytest.mark.ignore_stream('5.8', '5.9')]
+pytestmark = [pytest.mark.ignore_stream('5.8')]
 
 
-@pytest.mark.tier(0)
-def test_compute_migration_modal(appliance):
-    """Test takes an appliance and tries to navigate to migration page and open mapping wizard.
-
-    This is a dummy test just to check navigation.
-    # TODO : Replace this test with actual test.
-    """
-    infra_mapping_collection = appliance.collections.v2v_mappings
-    view = navigate_to(infra_mapping_collection, 'All')
-    assert view.is_displayed
-    view = navigate_to(infra_mapping_collection, 'Add')
-    assert view.is_displayed
+def test_migration_ui_enable_disable(appliance):
+    appliance.enable_migration_ui()
+    view = navigate_to(appliance.server, 'LoggedIn', wait_for_view=True)
+    nav_tree = view.navigation.nav_item_tree()
```

Comment:
**REQUIRED:** Here, we should not be navigating to 'LoggedIn' but 'Dashboard'. It is a proper _further_ destination. Same as below.